csharp/src/Ice/Connection.cs

```diff
@@ -710,7 +710,7 @@ public void AsyncRequestCanceled(OutgoingAsyncBase outAsync, System.Exception ex
                     {
                         if (kvp.Value == outAsync)
                         {
-                            if (ex is ConnectionTimeoutException)
+                            if (ex is ConnectionIdleException)
```

Comment:
Same issue there, ex is never ConnectionIdleException here.

Reply:
Fixed (removed)